src/client.c

```diff
@@ -68,6 +70,13 @@ PgDatabase *prepare_auth_database(PgSocket *client)
 		return NULL;
 	}
 
+    if (auth_db->admin) {
+        slog_error(client, "cannot use the reserved \"%s\" database as an auth_dbname", auth_db->dbname);
+        slog_warning(client, "set global \"auth_dbname\" parameter and define database in the database section");
```

Comment:
I do not think there is a need to provide a warning here, as the error message is clearly explaining what is wrong.

Reply:
Got it. I'll keep only the error statement

Reply:
Done